src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -124,12 +124,11 @@ class LocalBackend extends Backend with SharedFileSystem {
    * as some extra shell code for monitoring jobs
    */
   private def writeScript(backendCall: BackendCall, instantiatedCommand: String, containerRoot: Path) = {
-    val (_, scriptWriter) = backendCall.script.fileAndWriter
-    scriptWriter.writeWithNewline("#!/bin/sh")
-    scriptWriter.writeWithNewline(s"cd $containerRoot")
-    scriptWriter.writeWithNewline(instantiatedCommand)
-    scriptWriter.writeWithNewline("echo $? > rc")
-    scriptWriter.flushAndClose()
+    backendCall.script.appendLines(
+      "#!/bin/sh",
+      s"cd $containerRoot",
+      instantiatedCommand,
+      "echo $? > rc")
```

Comment:
I'll change it, but I just remembered how much I hate fighting with the IntellJ bugs regarding auto formatting & multiple line strings.